src/plugins/intel_cpu/src/nodes/experimental_detectron_roifeatureextractor.cpp

```diff
@@ -106,205 +102,45 @@ void pre_calc_for_bilinear_interpolate(
-
-        // We do average (integral) pooling inside a bin
-        const T count = static_cast<T>(roi_bin_grid_h * roi_bin_grid_w);  // e.g. = 4
-
-        // we want to precalculate indices and weights shared by all chanels,
-        // this is the key point of optimiation
-        std::vector<PreCalc<T>> pre_calc(
-                roi_bin_grid_h * roi_bin_grid_w * pooled_width * pooled_height);
-        pre_calc_for_bilinear_interpolate(
-                height,
-                width,
-                pooled_height,
-                pooled_width,
-                roi_bin_grid_h,
-                roi_bin_grid_w,
-                roi_start_h,
-                roi_start_w,
-                bin_size_h,
-                bin_size_w,
-                roi_bin_grid_h,
-                roi_bin_grid_w,
-                pre_calc);
-
-        for (int c = 0; c < channels; c++) {
-            int index_n_c = index_n + c * pooled_width * pooled_height;
-            const T* offset_bottom_data =
-                    bottom_data + (roi_batch_ind * channels + c) * height * width;
-            int pre_calc_index = 0;
-
-            for (int ph = 0; ph < pooled_height; ph++) {
-                for (int pw = 0; pw < pooled_width; pw++) {
-                    int index = index_n_c + ph * pooled_width + pw;
-
-                    T output_val = 0.;
-                    for (int iy = 0; iy < roi_bin_grid_h; iy++) {
-                        for (int ix = 0; ix < roi_bin_grid_w; ix++) {
-                            PreCalc<T> pc = pre_calc[pre_calc_index];
-                            output_val += pc.w1 * offset_bottom_data[pc.pos1] +
-                                          pc.w2 * offset_bottom_data[pc.pos2] +
-                                          pc.w3 * offset_bottom_data[pc.pos3] +
-                                          pc.w4 * offset_bottom_data[pc.pos4];
-
-                            pre_calc_index += 1;
-                        }
-                    }
-                    output_val /= count;
-
-                    top_data[index] = output_val;
-                }  // for pw
-            }  // for ph
-        }  // for c
-    });
-}
-
+void roiTargetLevels(const float* rois, int* levels, const size_t numRois, const int numLevels) {
+    constexpr float canonical_scale = 224.0f;
+    constexpr int canonical_level = 2;
 
-void redistribute_rois(const float* rois, int* level_ids,
-                       const int num_rois, const int levels_num) {
-    const float canonical_scale = 224.0f;
-    const int canonical_level = 2;
-
-    for (int i = 0; i < num_rois; ++i) {
+    for (size_t i = 0; i < numRois; ++i) {
         const float x0 = rois[4 * i + 0];
         const float y0 = rois[4 * i + 1];
         const float x1 = rois[4 * i + 2];
         const float y1 = rois[4 * i + 3];
 
-        int target_level = levels_num;
+        int target_level = numLevels;
```

Comment:
I think it is better to support the same code style naming like in original implementation.
For example, if in implementation lots of snake-case variable names, lets use this convention otherwise it will make new modification inconsistent with other code for this operator and over tha time it could become even more inconsistent

Reply:
Done